src/chainparams.cpp

```diff
@@ -282,12 +283,11 @@ class CRegTestParams : public CChainParams {
 
         UpdateActivationParametersFromArgs(args);
 
-        genesis = CreateGenesisBlock(1585958400, 0x0278eefd, 0x1d22ffff, 1, 200 * COIN);
+        genesis = CreateGenesisBlock(1585958400, 5, 0x207fffff, 1, 200 * COIN);
```

Comment:
actually you cannot have block hash more than 0x7xxx... as this will result in IsNegative check triggering